Styles

Inhaasap opened the pull request.

Inhaasap committed:
index.css

added fonts, updated index.css, added images folder, thema folder

Inhaasap committed:
updated index.css

Hatrison reviewed (commented):
Стільки малюнків нам не потрібно, для секції ревью стовідсотково (в нас динамічні відгуки та динамічні аватари). Також картинка на сторінці 404 інша (дивись макет). В папці account не потрібен малюнок календаря, він також динамічний. В папці свг індекс було б гарно прибрати (усі рівно від імпортів у компонентах не позбутися) або дописати, але це марна витрата часу. Також не вистачає iconLogout та не побачив плюсика при додаванні аватару.
Щодо теми, додай, будь ласка, ThemeProvider в index.js.
Дякую!

Inhaasap committed:
fixed

i deleted all images, i think i let each person add the img what does he/she needs, i was needed a few icons for myself.

Hatrison committed:
Added theme provider and redux state for it

Hatrison committed:
Refactored theme provider and created example of usage

Hatrison committed:
Merge branch 'styles' of https://github.com/Hatrison/GooseTrack into styles

Hatrison committed:
Improved exports

Hatrison committed:
Removed unused import
